.github/workflows/build_dev_documentation.yml

```diff
@@ -74,7 +81,7 @@ jobs:
           apt-get -y update && apt-get install -y libsndfile1-dev
 
           pip uninstall -y doc-builder
-          pip install git+https://github.com/huggingface/doc-builder -U
+          pip install git+https://github.com/huggingface/doc-builder@kit -U
```

Comment:
note: merge after https://github.com/huggingface/doc-builder/pull/94 is merged & remove `@kit`